### PR TITLE
add proxy-bypass-list as argv argument

### LINKED ISSUE #216642: Add --proxy-bypass-list support in argv.json
Add support for --proxy-bypass-list in the argv.json file so that we can provide a default proxy bypass list to VS Code users. 

Setting NO_PROXY doesn't work as a solution because it would either have to be set as a user / system environment variable thus impacting all applications, or we would have to rely on our VS Code users to explicitly pass --proxy-bypass-list as an argument to code on the command line each time they launch. Having the option to set this in argv.json would provide us with a way to package this in our default enterprise installation of VS Code so users would get this setting by default, but just for VS Code.

### PR DESCRIPTION
**Pull request by isidorn**
fixes #216642

### CODE REVIEW

File: src/vs/workbench/electron-sandbox/desktop.contribution.ts [hunk 1]
```diff
@@ -360,6 +360,10 @@ import { MAX_ZOOM_LEVEL, MIN_ZOOM_LEVEL } from 'vs/platform/window/electron-sand
 				type: 'boolean',
 				description: localize('argv.disableLcdText', 'Disables LCD font antialiasing.')
 			},
+			'proxy-bypass-list': {
+				type: 'string',
+				description: localize('argv.proxyBypassList', 'Bypass any specified proxy for the given semi-colon-separated list of hosts.')
```
Comment (deepak1556) [inline src/vs/workbench/electron-sandbox/desktop.contribution.ts:365]: Maybe add example of host format accepted in this arg, can use the example from https://github.com/electron/electron/blob/main/docs/api/command-line-switches.md#--proxy-bypass-listhosts

**Review by deepak1556 — COMMENTED** [review]
LGTM, minor nit

**Comment by isidorn** [comment]
Thanks for reviewing. I have reacted on your comment and added an example.

**Review by chrmarti — APPROVED** [review]
LGTM!

**Pull request merged**